### PR TITLE
Us ky franklin

### PR DESCRIPTION
**Pull request by olleicua**

### CODE REVIEW

File: recidiviz/tests/ingest/scrape/regions/us_ky_franklin/.#us_ky_franklin_scraper_test.py [hunk 1]
```diff
@@ -0,0 +1 @@
+root@e6ff5e5ac631.4785
```
Comment (colincadams) [inline recidiviz/tests/ingest/scrape/regions/us_ky_franklin/.#us_ky_franklin_scraper_test.py]: what editor do you use? I assume this file is one that it generated? If so let us know so we can set up our repo to ignore files of this form.

File: recidiviz/ingest/scrape/regions/us_ky_franklin/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+agency_name: $agency
```
Comment (colincadams) [inline recidiviz/ingest/scrape/regions/us_ky_franklin/manifest.yaml]: set the agency

File: recidiviz/tests/ingest/scrape/regions/us_ky_franklin/.#us_ky_franklin_scraper_test.py (hunk 1 above)
Comment (olleicua) [reply recidiviz/tests/ingest/scrape/regions/us_ky_franklin/.#us_ky_franklin_scraper_test.py]: my bad, those come from emacs. I removed them.

Comment (colincadams) [reply recidiviz/tests/ingest/scrape/regions/us_ky_franklin/.#us_ky_franklin_scraper_test.py]: great thanks! I just sent out #1083 so hopefully they'll all be ignored in the future!

File: recidiviz/ingest/scrape/regions/us_ky_franklin/manifest.yaml (hunk 2 above)
Comment (olleicua) [reply recidiviz/ingest/scrape/regions/us_ky_franklin/manifest.yaml]: agency set

**Comment by olleicua** [comment]
ping

**Pull request merged**